src/SourceBuild/tarball/patches/aspnetcore/0004-Don-t-override-BuildNodeJS-in-build.sh.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Logan Bussell <loganbussell@microsoft.com>
+Date: Mon, 15 Aug 2022 16:21:57 -0700
+Subject: [PATCH] Don't override BuildNodeJS in build.sh
+
+Issue: https://github.com/dotnet/aspnetcore/issues/43310
```

Comment:
Minor suggestion:  I've been trying to standardize around the `Backport:` header regardless if the link is an issue or PR.